src/formio-component.component.ts

```diff
@@ -141,4 +141,12 @@ export class FormioComponentComponent<T> extends Type implements OnInit {
         });
         return errors;
     }
+    allowMultiple(): boolean {
+        if(this.component.type=='select'){
```

Comment:
We should never do this within the base component. This is why we have inheritance, so that you can override the base class and do this within the Select component. Please fix.